.github/workflows/advanced.yml

```diff
@@ -355,16 +389,6 @@ jobs:
         run: |
           .github/scripts/toolchain/pack-runtime.sh
```

Comment:
I am confused, step is called Pack toolchain, however it packs the runtime.

Reply:
It's called "Pack runtime". Did you expand the collapsed code diff in the GitHub?